tests/api/test_ref_cmmnt_and_crrctn.py

```diff
@@ -0,0 +1,115 @@
+    with TestClient(app) as client:
+        ref1 = test_reference.json()
+        ref2 = create_test_reference2.json()
+        new_rcc = {"reference_curie_from": ref1,
+                   "reference_curie_to": ref2,
+                   "reference_comment_and_correction_type": "CommentOn"
+                   }
+        response = client.post(url="/reference_comment_and_correction/", json=new_rcc, headers=auth_headers)
+        yield TestRefComAndCorData(response, response.json(), ref1, ref2)
+
+
+class TestReferenceCommentAndCorrection:
+
+    def test_get_bad_rcc(self):
+        with TestClient(app) as client:
+            response = client.get(url="/reference_comment_and_correction/-1")
+            assert response.status_code == status.HTTP_404_NOT_FOUND
+
+    def test_bad_missing_args(self, test_reference, auth_headers): # noqa
+        with TestClient(app) as client:
+            xml = {"reference_curie_from": test_reference.json(),
+                   "reference_comment_and_correction_type": "CommentOn"
+                   }
+            response = client.post(url="/reference_comment_and_correction/", json=xml, headers=auth_headers)
+            assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+
+            xml = {'reference_curie_to': test_reference.json(),
+                   'reference_comment_and_correction_type': "CommentOn"
+                   }
+            response = client.post(url="/reference_comment_and_correction/", json=xml, headers=auth_headers)
+            assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+
+            xml = {'reference_curie_from': test_reference.json(),
+                   'reference_curie_to': test_reference.json()}
+            response = client.post(url="/reference_comment_and_correction/", json=xml, headers=auth_headers)
+            assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+
+    def test_create_rcc(self, db, test_ref_cc): # noqa
+        # check results in database
+        rcc_obj = db.query(ReferenceCommentAndCorrectionModel).join(
+            ReferenceModel, ReferenceCommentAndCorrectionModel.reference_id_from == ReferenceModel.reference_id).filter(
+            ReferenceModel.curie == test_ref_cc.ref_curie_from).one()
+        assert rcc_obj.reference_to.curie == test_ref_cc.ref_curie_to
+        assert rcc_obj.reference_comment_and_correction_type == "CommentOn"
+
+    def test_patch_rcc(self, db, test_ref_cc, auth_headers): # noqa
+        with TestClient(app) as client:
+            # swap to and from and change correction type
+            xml = {'reference_curie_from': test_ref_cc.ref_curie_to,
+                   'reference_curie_to': test_ref_cc.ref_curie_from,
+                   'reference_comment_and_correction_type': "ReprintOf"
+                   }
+            response = client.patch(url=f"reference_comment_and_correction/{test_ref_cc.new_rcc_id}",
```

Comment:
should this have a / between url=f" and reference_comment_and_correction ?  same with lines 83 85 86

Reply:
I will add the missing "/". I think that it should be there, at least to be consistent with the other calls